### DISCUSSION TITLE
export typebox helper function arrayOfKeys

### DISCUSSION THREAD

**Discussion by chunlampang** [feathersjs/feathers#3350] [Q&A]
I want to use this function, but it is not exported.

https://github.com/feathersjs/feathers/blob/a21b2cfcc5a8038b7782e66920d68cf29051c7d9/packages/typebox/src/index.ts#L56-L66

**Comment by daffl** [comment]
It was only really intended to be used internally (since it is unsafe). What are you trying to do?
